### PR TITLE
Fix pylint's redefined-builtin

### PR DESCRIPTION
**Pull request by marcospri**

### CODE REVIEW

File: h/accounts/schemas.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def get_blacklist():
         # can't load the file, then don't crash out, just log a warning about
         # the problem.
         try:
-            with open("h/accounts/blacklist", encoding="utf-8") as fp:
+            with codecs.open("h/accounts/blacklist", encoding="utf-8") as fp:
```
Comment (jon-betts) [inline h/accounts/schemas.py:32]: Oh god there's no way you'd spot this when reading the code. Also, is this any different from just `open("h/accounts/blacklist")` now that `utf-8` is the default?

File: h/models/document/_meta.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def __repr__(self):
         return "<DocumentMeta %s>" % self.id
 
 
-def create_or_update_document_meta(
+def create_or_update_document_meta(  # pylint:disable=redefined-builtin
     session, claimant, type, value, document, created, updated
 ):
```
Comment (jon-betts) [inline h/models/document/_meta.py:53]: We often do `type_` elsewhere?

File: h/accounts/schemas.py (hunk 1 above)
Comment (jon-betts) [inline h/accounts/schemas.py:32]: Oh god there's no way you'd spot this when reading the code. Also, is this any different from just `open("h/accounts/blacklist")` now that `utf-8` is the default?

File: h/models/document/_meta.py (hunk 2 above)
Comment (jon-betts) [inline h/models/document/_meta.py:53]: We often do `type_` elsewhere?

**Review by jon-betts — APPROVED** [review]
Yep this is definitely better.

I had one question about `codec.open`. How is that different from a conventional open in Python 3 now that everything is UTF-8 by default?

Also, I'd be totally happy to review files, or directories in one go. Fixing a particular type in one PR seems like it must have taken some effort for you to do.

File: h/accounts/schemas.py (hunk 1 above)
Comment (marcospri) [reply h/accounts/schemas.py:32]: They seem to return slightly different types but it would probably fine to use either here

**Pull request merged**